azure/packages/azure-client/src/AzureFunctionTokenProvider.ts

```diff
@@ -12,6 +12,11 @@ import { AzureMember } from "./interfaces";
  * Token Provider implementation for connecting to an Azure Function endpoint for
  * Azure Fluid Relay token resolution.
  */
+/**
+ * @deprecated 1.1.2, This API will be removed in 1.1.3
```

Comment:
Are these version numbers correct? Removing this API is a breaking change, so it should only occur in a major version increment, not a patch increment.

Reply:
For clarity, what @Josmithr is saying, we can deprecate in minor release. But actually removal can only happen in major release. Semver is "major.minor.patch". So if your deprecation will be published in next minor release (1.2.0) then the earliest this class can be deleted is 2.0.0.

Reply:
I'll update the version numbers accordingly.